wsmaster/codenvy-hosted-api-audit/src/main/java/com/codenvy/api/audit/server/AuditManager.java

```diff
@@ -123,6 +136,57 @@ public Path generateAuditReport() throws ServerException, ConflictException {
         return auditReport;
     }
 
+    /**
+     * Print license actions info in chronological order.
+     * @param auditReport
+     * @throws ServerException
+     */
+    private void processLicenseActions(Path auditReport) throws ServerException {
+        Map<Long, Runnable> actions = new TreeMap<>();
```

Comment:
There is Map used to sort licence actions automatically by timestamp.
Without Runnable it will be a little bit more verbose.